arrow-cast/src/display.rs

```diff
@@ -38,6 +38,22 @@ use lexical_core::FormattedSize;
 
 type TimeFormat<'a> = Option<&'a str>;
 
+/// Format for displaying decimals
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
+#[non_exhaustive]
+pub enum DecimalFormat {
+    /// Render decimals as JSON numbers, e.g. 12.34
+    Number,
+    /// Render decimals as JSON strings, e.g. "12.34"
+    String,
+}
+
+impl Default for DecimalFormat {
+    fn default() -> Self {
+        Self::Number
+    }
+}
```

Comment:
A bit shorter, but still explicit:
```suggestion
/// Format for displaying decimals
#[derive(Default, Debug, Copy, Clone, PartialEq, Eq, Hash)]
#[non_exhaustive]
pub enum DecimalFormat {
    /// Render decimals as JSON numbers, e.g. 12.34
    #[default]
    Number,
    /// Render decimals as JSON strings, e.g. "12.34"
    String,
}
```